SEPRTR/iocBoot/iocSEPRTR-IOC-01/st-daq.cmd

```diff
@@ -0,0 +1,16 @@
+## NI cDAQ-9185
+epicsEnvSet("CDAQER","cDAQ9185-1D195CFMod2")
+epicsEnvSet("CDAQAI","cDAQ9185-1D195CFMod3")
+epicsEnvSet("CDAQAO","cDAQ9185-1D195CFMod4")
+
+## input
+$(IFNOTDEVSIM) DAQmxConfig("R0", "$(CDAQAI)/ai0", 0, "AI","N=1000 F=1000") ## Kicker Volt
```

Comment:
Can the `N` and `F` values be supplied by the users as a macro so these can be changed on the fly? Knowing the data rate coming from the device is quite important, it may even be worth putting these in a PV to aid processing of the data.